modules/microshift-low-latency-config-yaml.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-config-yaml_{context}"]
+= Configuration kubelet parameters and values in {microshift-short}
+
+The first step in enabling low latency to a {microshift-short} cluster is to add configurations to the {microshift-short} `config.yaml` file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* You made a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+//anything else? do we need to create the KubeletConfiguration first? or does this config populate that for us?
+
+.Procedure
+
+. Add the kubelet configuration to the {microshift-short} `config.yaml` file
++
+.Example passthrough `kubelet` configuration
+[source,yaml]
+----
+# ...
+kubelet: <1>
+  cpuManagerPolicy: static
+  cpuManagerPolicyOptions:
+    full-pcpus-only: "true"
+  cpuManagerReconcilePeriod: 5s
+  memoryManagerPolicy: Static
+  topologyManagerPolicy: single-numa-node
+  reservedSystemCPUs: 0-1
+  reservedMemory:
+  - limits:
+      memory: 1100Mi <2>
+    numaNode: 0
+  kubeReserved:
+    memory: 500Mi
+  systemReserved:
+    memory: 500Mi
+  evictionHard: <3>
+    imagefs.available: "15%" <4>
+    memory.available: "100Mi" <5>
+    nodefs.available: "10%" <6>
+    nodefs.inodesFree: "5%" <7>
+  evictionPressureTransitionPeriod: 0s
+# ...
+----
+//How many footnotes do we want to include? Any others?
```

Comment:
I think these could be useful:
- cpuManagerPolicy
- memoryManagerPolicy
- full-pcpus-only

reservedSystemCPUs would be kind of must have as this is very important to dividing management workload and apps